python/pyspark/sql/column.py

```diff
@@ -430,6 +454,19 @@ def getField(self, name: Any) -> "Column":
 
         .. versionadded:: 1.3.0
 
+        Parameters
+        ----------
+        name
+            a literal value, or a :class:`Column` expression.
+            The result will only be true at a location if field matches in the Column.
```

Comment:
@dcoliversun Could you clarify what you mean here and in other places where you use similar phrasing?

Reply:
I refer to [isin](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.isin.html) in pandas document. It phrases that Column is selected when field in the column matches conditions.